---
 Makefile |    4 ++++
 1 file changed, 4 insertions(+)

Index: linux-3.19-ck1/Makefile
===================================================================
--- linux-3.19-ck1.orig/Makefile	2015-02-27 17:03:42.250631758 +1100
+++ linux-3.19-ck1/Makefile	2015-02-27 17:03:42.248631758 +1100
@@ -10,6 +10,10 @@ NAME = Diseased Newt
 # Comments in this file are targeted only to the developer, do not
 # expect to learn how to build the kernel reading this file.
 
+CKVERSION = -ck1
+CKNAME = BFS Powered
+EXTRAVERSION := $(EXTRAVERSION)$(CKVERSION)
+
 # Do not use make's built-in rules and variables
 # (this increases performance and avoids hard-to-debug behaviour);
 MAKEFLAGS += -rR
