---
 Makefile |    4 ++++
 1 file changed, 4 insertions(+)

Index: linux-4.0-ck1/Makefile
===================================================================
--- linux-4.0-ck1.orig/Makefile	2015-04-16 14:14:38.558700723 +1000
+++ linux-4.0-ck1/Makefile	2015-04-16 14:14:38.556700735 +1000
@@ -10,6 +10,10 @@ NAME = Hurr durr I'ma sheep
 # Comments in this file are targeted only to the developer, do not
 # expect to learn how to build the kernel reading this file.
 
+CKVERSION = -ck1
+CKNAME = BFS Powered
+EXTRAVERSION := $(EXTRAVERSION)$(CKVERSION)
+
 # Do not use make's built-in rules and variables
 # (this increases performance and avoids hard-to-debug behaviour);
 MAKEFLAGS += -rR
